src/app/personside/BegrensetTilgangSide.tsx

```diff
@@ -2,24 +2,80 @@ import * as React from 'react';
+import { useRestResource } from '../../rest/consumer/useRestResource';
+import { useFødselsnummer } from '../../utils/customHooks';
+import { usePostResource } from '../../rest/consumer/usePostResource';
+import { OpprettOppgaveRequest } from '../../models/meldinger/oppgave';
+import { opprettOppgaveActionCreator } from '../../redux/restReducers/meldinger/opprettOppgave';
+import { useDispatch } from 'react-redux';
+import styled from 'styled-components/macro';
+import Ekspanderbartpanel from 'nav-frontend-ekspanderbartpanel';
+import { useState } from 'react';
 
 interface BegrensetTilgangProps {
     person: BegrensetTilgang;
 }
 
+const Wrapper = styled.div`
+    display: flex;
+    flex: auto;
+    flex-direction: column;
+    flex-grow: 0;
+    > *:first-child {
+        margin-bottom: 1rem;
+    }
+`;
+
+function OpprettOppgaveAvvistTilgang() {
+    const gsakTemaResource = useRestResource(resources => resources.oppgaveGsakTema);
+    const gsakTema = gsakTemaResource?.data;
+    const fnr = useFødselsnummer();
+    const innloggetSaksbehanderResource = useRestResource(resources => resources.innloggetSaksbehandler);
+    const innloggetSaksbehandler = innloggetSaksbehanderResource?.data;
+    const opprettOppgaveResource = usePostResource(resources => resources.opprettOppgave);
+    const dispatch = useDispatch();
+    const opprettOppgave = (request: OpprettOppgaveRequest) => dispatch(opprettOppgaveActionCreator(request));
+
+    const [apen, setApen] = useState(false);
+
+    const lukk = () => {
+        setApen(!apen);
+    };
+
+    if (!gsakTema || !innloggetSaksbehandler) {
+        return <AlertStripeInfo>Kunne ikke vise opprett oppgave panel</AlertStripeInfo>;
+    }
+
+    return (
+        <Ekspanderbartpanel tittel={'Opprett oppgave'} apen={apen} onClick={() => setApen(!apen)}>
+            <OppgaveSkjemaSkjermetPerson
+                gsakTema={gsakTema}
+                gjeldendeBrukerFnr={fnr}
+                innloggetSaksbehandler={innloggetSaksbehandler}
+                opprettOppgaveResource={opprettOppgaveResource}
+                opprettOppgave={opprettOppgave}
+                lukkPanel={lukk}
+            />
+        </Ekspanderbartpanel>
+    );
+}
 function BegrensetTilgangSide({ person }: BegrensetTilgangProps) {
-    const erEgenAnsatt = BegrensetTilgangTyper[person.begrunnelse] === BegrensetTilgangTyper.EgenAnsatt;
+    const erEgenAnsatt = person.begrunnelse === BegrensetTilgangTyper.EgenAnsatt;
     return (
         <FillCenterAndFadeIn>
-            <AlertStripe type="advarsel">
-                <BegrensetTilgangBegrunnelse begrunnelseType={person.begrunnelse} />
-                {visSikkerhetstiltak(person.sikkerhetstiltak, erEgenAnsatt)}
-            </AlertStripe>
+            <Wrapper>
+                <AlertStripe type="advarsel">
+                    <BegrensetTilgangBegrunnelse begrunnelseType={person.begrunnelse} />
+                    {visSikkerhetstiltak(person.sikkerhetstiltak, erEgenAnsatt)}
```

Comment:
bruker me peson.sikkerhetstiltak som generisk innhold for kvifor ein får ikke tilgang?

Reply:
opprett oppgave må støtte kode 6/kode 7 og geografisk avvisning, ikkje berre egenAnsatt.

Reply:
Ja den støtter alle :)

Reply:
`erEgenAnsatt` er kun for å vise riktig type overskrift og melding. Fikk kommentarer på at Egen Ansatt ikke er et sikkerhetstiltak